gw_eccentricity/eccDefinition.py

```diff
@@ -271,10 +289,16 @@ def __init__(self, dataDict, num_orbits_to_exclude_before_merger=2,
         if not np.allclose(self.t_diff, self.t_diff[0]):
             raise Exception("Input time array must have uniform time steps.\n"
                             f"Time steps are {self.t_diff}")
-        # get amplitude, phase and omega of 22 mode
-        self.amp22 = self.dataDict["amplm"][(2, 2)]
-        self.phase22 = self.dataDict["phaselm"][(2, 2)]
-        self.omega22 = self.dataDict["omegalm"][(2, 2)]
+        # get amplitude, phase, omega to be used for eccentricity measurement.
+        # For precessing systems, even in the coprecessing frame, the (2, 2)
+        # mode quantities show some oscillations due to the precession
+        # effect. To reduce these oscillations further, we use a combination of
+        # (2, 2) and (2, -2) mode in the coprecessing frame to define new
```

Comment:
Instead of saying "a combination", which is vague, just point to the equation in the surrogate paper.

Reply:
Done